.danger/dangerfile-lint.ts

```diff
@@ -0,0 +1,10 @@
+import {danger, fail, message, warn} from 'danger'
+
+const reporter = require("danger-plugin-lint-report")
```

Comment:
```suggestion
const reporter = require("../src")
```

If you want to use the current version